sentry/src/main/java/io/sentry/SentryTransaction.java

```diff
@@ -28,7 +28,9 @@
    * A hub this transaction is attached to. Marked as transient to be ignored during JSON
    * serialization.
    */
-  private @NotNull final transient IHub hub;
+  private final transient @NotNull IHub hub;
+
+  private final transient @NotNull SpanContext context;
```

Comment:
```suggestion
  private final transient @NotNull SpanContext contexts;
```

Reply:
Wasn't the goal to move the name to `context`? Since `SpanContext`?

Reply:
no goal, but I'd rather prefer the keep consistency, either the ctor param or the global field should rename, so they match.
it's a nitpick but why not naming the same?

Reply:
renamed to "context"